framework/src/utils/FormattedTable.C

```diff
@@ -50,6 +51,68 @@ dataLoad(std::istream & stream, FormattedTable & table, void * context)
   loadHelper(stream, table._headers_output, context);
 }
 
+template <>
+void
+dataStore(std::ostream & stream, std::shared_ptr<TableValueBase> & value_base, void * context)
+{
+  value_base->store(stream, context);
+}
+
+template <>
+void
+dataLoad(std::istream & stream, std::shared_ptr<TableValueBase> & value_base, void * context)
+{
+  std::string type;
+  dataLoad(stream, type, context);
+  if (type == typeid(bool).name())
```

Comment:
Look into std::is_same for these ifs

Reply:
We need to store the type somehow has restartable data. My way was to just store the typeid string. Is there another way we can store the type?

Reply:
I think if we were able to use `is_same`, we wouldn't need these if statements.